### PR TITLE
collection of plugin cleanups

### PR DESCRIPTION
**Pull request by KyleSanderson**
Just doing drive-bys on first-party plugins.

### CODE REVIEW

File: plugins/basebans.sp [hunk 1]
```diff
@@ -86,7 +86,7 @@ public void OnPluginStart()
 	}
 }
 
-public void OnMapStart()
+public void OnConfigsExecuted()
```
Comment (asherkin) [inline plugins/basebans.sp]: `LoadBanReasons` doesn't rely on anything from a config file, `OnMapStart` is a better choice.

File: plugins/rockthevote.sp [hunk 2]
```diff
@@ -99,47 +92,40 @@ public void OnMapStart()
...
 }
 
 public void OnClientDisconnect(int client)
 {
-	if(IsFakeClient(client))
-		return;
+	g_Voted[client] = false;
 	
-	if(g_Voted[client])
+	if (g_Voted[client])
```
Comment (asherkin) [inline plugins/rockthevote.sp]: This will never be hit.

Comment (KyleSanderson) [reply plugins/rockthevote.sp]: Great catch, thanks

File: plugins/basebans.sp (hunk 1 above)
Comment (KyleSanderson) [reply plugins/basebans.sp]: g_BanReasonsPath is a config file :-)

**Pull request merged**